templates/supplier/not_allowed.html

```diff
@@ -0,0 +1 @@
+<p>Not Allowed!</p>
```

Comment:
Either throw the user into an error page (not recommended), redirect him to a different page (with details regarding why he is redirected), or redirect him to a page like this `Not Allowed` one - but allow him to go back (is the navbar available for him?)

Reply:
Also in its current state I think this page should be available all over the app and not only through supplier's templates

Reply:
will work on the nav bar today.


> Also in its current state I think this page should be available all over the app and not only through supplier's templates

agree